data/commodities.txt

```diff
@@ -622,6 +746,7 @@ trade
 		"custom pens"
 		"designer clothing"
 		"designer suits"
+		"diamonds"
```

Comment:
Could also add `gemstones` and other precious stones to this list, although it's by no means necessary.